src/pages/workspaces/workspace/Container.js

```diff
@@ -3,64 +3,74 @@ import { Component, Fragment } from 'react'
 import { a, div, h, hh } from 'react-hyperscript-helpers'
 import { contextBar, TopBar } from 'src/components/common'
 import { breadcrumb, icon } from 'src/components/icons'
-import * as Ajax from 'src/libs/ajax'
 import * as Nav from 'src/libs/nav'
 import * as Style from 'src/libs/style'
+import WorkspaceDashboard from 'src/pages/workspaces/workspace/Dashboard'
 import WorkspaceData from 'src/pages/workspaces/workspace/Data'
+import WorkspaceNotebooks from 'src/pages/workspaces/workspace/Notebooks'
 
 
 const navSeparator = div({
-  style: {
-    background: 'rgba(255,255,255,0.15)', width: 1, height: '3rem',
-    flexShrink: 0
-  }
+  style: { background: 'rgba(255,255,255,0.15)', width: 1, height: '3rem', flexShrink: 0 }
 })
 
-const tabBaseStyle = { maxWidth: 140, flexGrow: 1, color: Style.colors.textFadedLight }
+const tabBaseStyle = {
+  maxWidth: 140, flexGrow: 1, color: Style.colors.textFadedLight, textDecoration: 'none'
+}
 
 const tabActiveStyle = _.defaults({
-  backgroundColor: 'rgba(255,255,255,0.15)',
-  color: null,
-  lineHeight: 'calc(3.5rem - 4px)',
+  backgroundColor: 'rgba(255,255,255,0.15)', color: 'unset', lineHeight: 'calc(3.5rem - 4px)',
   borderBottom: `4px solid ${Style.colors.secondary}`
 }, tabBaseStyle)
 
-const navTab = (name, isActive = false) => {
-  return h(Fragment, [
-    div({ style: isActive ? tabActiveStyle : tabBaseStyle }, name),
-    navSeparator
-  ])
-}
-
 const navIcon = shape => {
   return icon(shape, { size: 22, style: { opacity: 0.65, paddingRight: '1rem' } })
 }
 
+const tabComponents = {
+  dashboard: WorkspaceDashboard,
+  notebooks: WorkspaceNotebooks,
+  data: WorkspaceData,
+  jobs: () => div('when we get to it'),
+  history: () => div('okay, this is _data_ history, whatever that means'),
+  tools: () => div('These are totally Not Method Configs')
```

Comment:
People are looking at this UI. I'm inclined to think we should have nice-looking placeholders here (sans snark ;) or remove the tabs until we've implemented something.